packages/react-ui/src/widgets/CategoryWidgetUI.js

```diff
@@ -363,11 +369,6 @@ function CategoryWidgetUI(props) {
     }
   }, [animation, sortedData]);
 
-  const getCategoriesCount = useCallback(() => {
-    const blocked = blockedCategories.length;
-    return blocked ? data.length - blocked : data.length - maxItems;
-  }, [data, maxItems, blockedCategories]);
-
   // Separated to simplify the widget layout but inside the main component to avoid passing all dependencies
   const CategoryItem = (props) => {
```

Comment:
Any refactor here, moving 'internal' components to different files would be really appreciated

Reply:
Great, I'll add it to my backlog 👌